src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/StructuralVariationDiscoveryPipelineSparkIntegrationTest.java

```diff
@@ -74,8 +79,15 @@ public String toString() {
         final File tempDirNew = BaseTest.createTempDir("new");
         tempDirNew.deleteOnExit();
         Files.createDirectories(Paths.get(tempDirNew.getAbsolutePath()+"/fastq"));
-        tests.add(new Object[]{new StructuralVariationDiscoveryPipelineSparkIntegrationTest.StructuralVariationDiscoveryPipelineSparkIntegrationTestArgs(SVIntegrationTestDataProvider.TEST_BAM_NEW, SVIntegrationTestDataProvider.KMER_KILL_LIST, SVIntegrationTestDataProvider.ALIGNER_INDEX_IMG, tempDirNew.getAbsolutePath())});
-
+        tests.add(new Object[]{
+                new StructuralVariationDiscoveryPipelineSparkIntegrationTest.StructuralVariationDiscoveryPipelineSparkIntegrationTestArgs(
+                        SVIntegrationTestDataProvider.TEST_BAM_NEW,
```

Comment:
What was the reason the test bam is called "new"? I know we replaced the test bam but the old one is gone right? Can we remove the "new'?

Reply:
Will change, I think this was just an artifact of when we replaced it.